docs/imputation.md

```diff
@@ -671,3 +679,24 @@ This command takes three parameters:
 * `--db-path` - Path to the directory containing the TileDB instances.
 * `--threads` - Number of threads for use by the TileDB loading
   procedure.
+
+### Create g.vcf files
```

Comment:
Add "(optional)" after title for extra clarity

Reply:
done